modules/deleting-a-connection-type.adoc

```diff
@@ -0,0 +1,32 @@
+:_module-type: PROCEDURE
+
+[id="deleting-a-connection-type_{context}"]
+= Deleting a connection type
+
+As an {productname-short} administrator, you can delete a connection type that you or another administrator created.
+
+ifdef::upstream[]
+Note that you cannot delete the connection types that are pre-installed with {productname-short}. Instead, you have the option of disabling them so that they are not visible to your users, as described in link:{odhdocshome}/managing-odh/#enabling-a-connection-type_resource-mgmt[Enabling a connection type].
+endif::[]
+ifndef::upstream[]
+Note that you cannot delete the connection types that are pre-installed with {productname-short}. Instead, you have the option of disabling them so that they are not visible to your users, as described in link:{rhoaidocshome}{default-format-url}/managing_openshift_ai/managing-connection-types#enabling-a-connection-type_resource-mgmt[Enabling a connection type].
+endif::[]
+
+
+.Prerequisites
+* You have logged in to {productname-short} as a user with {productname-short} administrator privileges. 
```

Comment:
The connection type must exist and must not be part of the pre-installed connection type (which you are unable to delete)